orttraining/orttraining/core/session/training_session.cc

```diff
@@ -287,7 +287,7 @@ Status TrainingSession::ConfigureForTraining(
   // derive actual set of weights to train
   std::unordered_set<std::string> weight_names_to_train =
       !filtered_config_weight_names_to_train.empty()
-          ? filtered_config_weight_names_to_train
+          ? trainable_initializers
```

Comment:
filtered_config_weight_names_to_train?

Reply:
changed this by intention, It should be trainable_initializers. This bug exists for a while. trainable_initializers contains the updated intializers (after megatron partition). We use "filtered_config_weight_names_to_train.empty()" to check whether this logic is started from C++ frontend or PyTorch fronentd.